pages/installation/coolify.mdx

```diff
@@ -40,7 +44,12 @@ services:
       CLOUDFLARE_BUCKET_URL: "https://QhcMSXQyPuMCRpSQcSYdEuTYgHeCXHbu.r2.cloudflarestorage.com/"
       CLOUDFLARE_REGION: "auto"
 
-      # These defaults are probably fine, but if you change your user/password, update it in the 
+      # These are totally fake values, you must change them with your own Resend settings.
+      RESEND_API_KEY: "re_QhcMSXQyP_CRpSQcSYdEuTYgHeCX"
```

Comment:
I suggest leaving this blank, allowing for automatic user activation

Reply:
I mean, leaving this out entirely

Reply:
It still wants to confirm by mail, dont know why